### DISCUSSION TITLE
Using ts-node with a fixed version of @types/node

### DISCUSSION THREAD

**Discussion by dplaton** [TypeStrong/ts-node#1749] [Q&A] [answered]
Hey everyone,

We are working in a monorepo that uses RushJS w/ pnpm. We have a policy of only having dependencies with fixed versions (i.e. no carets and no *). The problem is that when we're trying to use `ts-node v10.0.0` we get the following error:

``` 
ts-node@10.7.0 requires a peer of @types/node@* but none was installed
```
We do have dependencies to `'@types/node': 16.11.7`, but it seems that the package wants only the version specified with `*` . Is there a way around that? 

Thanks a lot and have a nice day

**Comment by cspotcode** [comment, marked as answer]
The asterisk dependency is compatible with all possible versions, we do this so that you can use whatever fixed version you want. This is likely a package manager issue, I recommend opening an issue with your package manager.

**Reply by dplaton** [reply]
Thanks for your reply, I'll take it up with the pnpm folks.

Have a nice day!
